doc/py_tutorials/py_setup/py_setup_in_ubuntu/py_setup_in_ubuntu.markdown

```diff
@@ -25,7 +25,7 @@ This method serves best when using just for programming and developing OpenCV ap
 Install package [python-opencv](https://packages.ubuntu.com/trusty/python-opencv) with following command in terminal (as root user).
 
 ```
-$ sudo apt-get install python-opencv
+$ sudo apt-get install python3-opencv
```

Comment:
There is another "python-opencv" few lines above with broken link (perhaps it make sense to update it to: https://packages.ubuntu.com/focal/python3-opencv )

Also there is a note above which probably should be updated too:

> Below steps are tested for Ubuntu 16.04 and 18.04 (both 64-bit).